### PR TITLE
Portfolio contact form

### PR DESCRIPTION
**Pull request by Booze33**
CONTACT-SECTION
-Created a new pull request for the branch
-Worked with my partner on the branch,
-Added the contact form to the portfolio,
-Checked and fixed all errors,
-Sent for review.

### CODE REVIEW

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @Booze33, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project that does not have a descriptive pull request summary describing the changes implemented in the pull request. Kindly highlight the things you implemented in your pull request summary.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -132,5 +132,29 @@ <h3>LETS CONNECT</h3>
         </ul>
     </div>
 </section>
+<section class="contact-form">
+    <h1>Contact me</h1>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form method="" action="">
```
Comment (souad988) [inline index.html:138]: kindly fill the method and action attributes as requested in this project requirements.

File: index.html [hunk 2]
```diff
@@ -132,5 +132,29 @@ <h3>LETS CONNECT</h3>
...
+                <input type="email" id="email" name="user-email" placeholder="Email" maxlength="30" required>
+                </label>
+            </li>
+            <li>
+                <label for="msg">
+                <textarea id="message" name="user_message" maxlength="500">Write your message here...</textarea>
+                </label>
+            </li>
+        </ul>
+        <button type="submit">Get in touch</button>
```
Comment (souad988) [inline index.html:156]: good job on implementing this part , Only one small issue to mention here your button has a different padding and font size , please make sure to replicate figma design as you can.
**your button:**

button padding student temp1

**figma design:**

button padding temp1

File: style.css [hunk 3]
```diff
@@ -538,4 +570,59 @@
...
+    font-family: var(--font-family);
+    font-size: 17px;
+    font-weight: 400;
+    line-height: 20px;
+    width: 327px;
+    height: 400px;
+  }
+
+  #name,
+  #email {
```
Comment (souad988) [inline style.css:597]: kindly add a padding left to all placeholder text in your inputs 😊
**your design:**

placeholder padding left  student

**figma design:**

placeholder padding left

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Booze33 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

212385281-4797c58a-d358-423a-aa7c-832c22203865


## Highlights
✔No linter errors
✔Very good and easy to understand commit messages 💪.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Booze33,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by Booze33** [comment]
Passed the review.
-close pull request

**Pull request merged**